packages/flutter/lib/src/widgets/title.dart

```diff
@@ -11,12 +11,15 @@ import 'framework.dart';
 /// A widget that describes this app in the operating system.
 class Title extends StatelessWidget {
   /// Creates a widget that describes this app to the operating system.
+  ///
+  /// If [title] is null, then the title will be set to the empty string.
   Title({
     Key key,
-    this.title,
+    String title,
     this.color,
     @required this.child,
   }) : assert(color == null || color.alpha == 0xFF),
+       title = title ?? '',
```

Comment:
the way we normally do that is we say that `title` has a default value of `''`, and we assert it's not null, and then we propagate that default to any other widgets that pass through.

Reply:
Yes, I understand, and that makes sense that you want to fail if someone passes a null explicitly, but what I was concerned about was breaking all apps that didn't supply a value, since if you don't supply a value to the MaterialApp, it'll supply a null to this function.  Should I add defaults and asserts to all of the apps that use this widget too? Note that it won't help people who have made their own Apps or MaterialApp subclasses.

Reply:
Ahh, actually I guess this is what you meant by the propagation part.  OK, will do.